### ISSUE TITLE
Gallery page optimization and "featured" mismatch

### ISSUE THREAD

**Issue by jywarren** [publiclab/mapknitter#842]
The new gallery page is great: https://mapknitter.org/gallery

However, i noticed it seems to be listing non-featured maps: 

![image](https://user-images.githubusercontent.com/24359/61071094-380db380-a3de-11e9-96f0-19a45a9bb94b.png)

Also, it takes a long time to load, so I wonder if it needs some query optimization? Many queries take 10-20 seconds:

![image](https://user-images.githubusercontent.com/24359/61071137-52479180-a3de-11e9-8900-f6afb3969d25.png)

https://oss.skylight.io/app/applications/ArYnJAb3VUC9/1562843760/6h/endpoints/FrontUiController%23gallery?responseType=html

Maybe we should just `limit()` this query to one result? 

https://github.com/publiclab/mapknitter/blob/ab7a964266321b84cf3352949ecd468f2854e946/app/views/front_ui/_maps.html.erb#L6

![image](https://user-images.githubusercontent.com/24359/61071200-7905c800-a3de-11e9-8d6e-f9a86ceac525.png)

I'll try that in #841.

**Comment by jywarren** [comment]
OK, tried this: 005d785d453bf965d254ddb3e96935c031543f49

**Comment by jywarren** [comment]
@cesswairimu would you be able to take a look at why it says "featured" but it seems to be showing all maps? Thank you!

**Comment by cesswairimu** [comment]
Hey Jeff, sure I believe we are still using this method https://github.com/publiclab/mapknitter/blob/main/app/models/map.rb#L86-L91 so it will show the maps that have atleast one image. I guess we need to update this to use the "featured" tag that you added?

**Comment by cesswairimu** [comment]
here is the helper method https://github.com/publiclab/mapknitter/blob/main/app/helpers/front_ui_helper.rb#L10-L13. If this seems right let me know and I will fix it. Thanks

**Comment by jywarren** [comment]
Hm. So would the current query be moved to the "All" tab, or maybe the word
"featured" is now being used for 2 things -- one for maps with at least one
image, and 2 for maps with the "featured" tag? So maybe... we need to think
on how to disambiguate. What do you think? Do I understand correctly?
Thanks, Cess!

**Comment by cesswairimu** [comment]
Yeah that's correct, The "all" tab currently is showing all the maps including those with no images yet
